lib/components/mixins/resizable.dart

```diff
@@ -1,15 +1,15 @@
-import 'dart:ui';
+import '../../vector2_extension.dart';
 
 /// Useful mixin to add to your components if you want to hold a reference to the current screen size.
 ///
 /// This mixin implements the resize method in order to hold an updated reference to the current screen [size].
 /// Also, it updates its [children], if any.
 class Resizable {
   /// This is the current updated screen size.
-  Size size;
+  Vector2 size;
 
   /// Implementation provided by this mixin to the resize hook.
-  void resize(Size size) {
+  void resize(Vector2 size) {
     this.size = size;
     resizableChildren().where((e) => e != null).forEach((e) => e.resize(size));
```

Comment:
I just realized this could be simplified greatly `resizableChildren().forEach((e) => e?.resize(size));`